docs/batch.rst

```diff
@@ -224,6 +227,9 @@ convert it to a OCRed PDF in ``/output/``. The parameters to this image are:
     "``-v <path to files to convert>:/input``", "Files placed in this location will be OCRed"
     "``-v <path to store results>:/output``", "This is where OCRed files will be stored"
     "``-e OCR_OUTPUT_DIRECTORY_YEAR_MONTH=1``", "This will place files in the output in {output}/{year}/{month}/{filename}"
+    "``-e OCR_ON_SUCCESS_DELETE=1``", "This will delete the input file if the exit code is 0 (OK)"
+    "``-e OCR_DESKEW=1``", "This will enable deskew for crooked PDFs"
```

Comment:
I kind of agree with having an option to deskew baked into the script since it's one of the most desirable to have, but there are tons of options in ocrmypdf and we don't want to set up environment vars for each. I think the idea should be that you take this script and customize it.

Reply:
Shouldn't there be something like in`webservice.py` where we could pass all arguments to the main program by calling it through 
```python
    cmd_args = [arg for arg in os.getenv('OCR_OPTIONS')]
    ocrmypdf_args = ["ocrmypdf", *cmd_args, file_path, output_path]
    proc = run(ocrmypdf_args, stdout=PIPE, stderr=PIPE, encoding="utf-8")
```